CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
 
+* Moved **generate-yml-from-python** to content docs.
```

Comment:
remove this

Reply:
*Validate Changelog* step will fail if I do, will you force merge if so?